src/components/structures/UserSettings.js

```diff
@@ -363,7 +363,23 @@ module.exports = React.createClass({
         */
         ];
 
+        var themes = [
```

Comment:
what is this? what are the fields for? why is it `var themes` here rather than `const THEMES` at the top-level?